src/main/scala/vsys/api/http/TransactionsApiRoute.scala

```diff
@@ -172,15 +172,27 @@ case class TransactionsApiRoute(
     pathPrefix(Segment) { address =>
       Address.fromString(address) match {
         case Left(e) => complete(ApiError.fromValidationError(e))
-        case Right(a) =>
-          complete(Json.arr(JsArray(state.activeLeases().flatMap(state.transactionInfo)
-              .map(a => (a._1,a._2,a._2.transaction))
+        case Right(a) => {
+          if(stateSettings.txTypeAccountTxIds){
+            val txNum = state.txTypeAccTxLengths(TransactionType.LeaseTransaction, a)
+            val txIds = state.txTypeAccountTxIds(TransactionType.LeaseTransaction, a, txNum, 0)
+            complete(Json.arr(JsArray(txIds._2
+              .flatMap(state.transactionInfo)
```

Comment:
`txIds._2.flatMap(state.transactionInfo)` is already implemented in `txTypeAccountTransactions`